rosidl_core_generators/CHANGELOG.rst

```diff
@@ -0,0 +1,45 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Maybe. My thinking was that this package was simply renamed, so it might be worth keeping the changelog here. However, considering we want to move this to a new repository I think starting a fresh changlog is okay too.

Reply:
After realizing the changelog doesn't contain much content, I'll probably remove this in the new repository

Reply:
> Maybe. My thinking was that this package was simply renamed, so it might be worth keeping the changelog here.

Ah okay, that seems fine.
It's a metapackage though, so there's not much stuff of value in the changelogs.

If we keep the old logs, we should also start from the next version number.